examples/chip-tool/commands/common/Command.h

```diff
@@ -0,0 +1,71 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#ifndef __CHIPTOOL_COMMAND_H__
+#define __CHIPTOOL_COMMAND_H__
+
+#include <vector>
+
+#include <controller/CHIPDeviceController.h>
+#include <core/CHIPError.h>
+#include <support/CHIPLogging.h>
+
+using namespace ::chip;
```

Comment:
Can we avoid using directives ("using namespace") in headers?

Reply:
Fixed.